src/Components/Header/NavLinks.js

```diff
@@ -0,0 +1,21 @@
+import NavLink from "./NavLink";
+import "./NavLinks.css";
+
+const getNavLinksTemplate = function (links) {
```

Comment:
this function could be added to helper

Reply:
Yes it can be, I still have doubt related to it. we will discuss it